recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -0,0 +1,31 @@
+# Avoid redundant runtime dependency on python3-core
+RDEPENDS:${PN}:remove:class-target = " ${PYTHON_PN}-core" 
+
+require xen-source.inc
+
+FILES:${PN}-test = "\
+    ${libdir}/xen/bin/test-xenstore \
+    ${libdir}/xen/bin/test-resource \
+    ${libdir}/xen/bin/test-paging-mempool\
```

Comment:
What is this and where it will go? rootfs?

Reply:
If I not mistaken it doesn't use now. This is for package xen-tools-test. That package is not included to final image

Reply:
Removed